springUsers/src/main/java/com/example/springUsers/Exceptions/CustomAppException.java

```diff
@@ -0,0 +1,14 @@
+package com.example.springUsers.Exceptions;
+
+import org.springframework.http.HttpStatus;
+
+import java.util.Date;
+
+public class  CustomAppException extends Exception {
+
+    private String message;
+
+    public CustomAppException(String message) {
```

Comment:
This custom exception doesn't do anything more than just a plain old Exception unless it's handled a different way, which it isn't.